packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -101,13 +101,17 @@ - (void)pickImageWithPHPicker:(int)maxImagesAllowed API_AVAILABLE(ios(14)) {
 }
 
 - (void)pickImageWithUIImagePicker {
```

Comment:
We should just not have this method. I really don't like the "put arguments in an ivar and read them later design; the bugs you are fixing here are exactly the sort of problem that kind of indirect system in notorious for. We should require the explicit argument, and over time completely eliminate `arguments` as an ivar.

AFAICT the calls are all from a context where we already have the arguments so this trampoline method shouldn't be needed.

Reply:
Removed this method as suggested.